Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/HeaderBasedAuthorization.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class HeaderBasedAuthorization : StoredAuthorization
+    {
+        public string key = "";
```

Comment:
Waarom zijn deze public? Ik zou verwacht hebben dat dit geencapsuleerd is in deze class of zijn kindclasses, als externe classes hiermee doen dan vraag ik mij af of dat niet een functie in deze value object moet zijn of zijn kindclasse

Reply:
Na het lezen van de rest van de code zie ik dat dit nodig is voor de 3D Tiles handler, niet sjiek maar wel noodzakelijk voor nu; maar dit een readonly property maken helpt wel